packages/web-data/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@vertex-protocol/web-data",
+  "version": "0.34.2",
+  "description": "",
+  "author": "Frank Jia <frank@vertexprotocol.com>",
+  "homepage": "",
+  "bugs": {
+    "url": ""
+  },
+  "license": "ISC",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "source": "./src/index.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vertex-protocol/vertex-sdk",
+    "directory": "packages/web-data"
+  },
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "build": "tsc -b",
+    "dev": "tsc -w"
+  },
+  "dependencies": {
+    "d3-format": "^3.1.0"
+  },
+  "peerDependencies": {
+    "@tanstack/react-query": "^5",
```

Comment:
More permissive versioning here for consumers. `devDependencies` is what's required for local dev on the SDK